content/reviewers_response.md

```diff
@@ -0,0 +1,642 @@
+
+**Our journal-based approach is still required as cosine/euclidean distance does not combat the issue of journal overrepresentation.
+We control for this issue by calculating a centroid for each journal.
+This centroid is constructed by taking the average of every published document present within each journal.
+This way, every journal will have an equal representation for our classifier.
+The shift in distribution could affect our paper-paper classifier which is why we provided top 10 results instead of best paper.
+Another way to combat this issue is to use a technique called orthogonal procrustes, which is a linear algebra techique designed to align two vector spaces.**
+
+> I love the web application!
+
+**We are happy you like the application.**
+
+> No statistics are given for the enrichments in Figure 1B-E.
+
+**We report each token with a 95% confidence interval (CI). 
+Our frequency counts are sufficiently large that all calculated p-values are approximately or exactly zero.**
+
+> I would welcome a supplemental analysis, that removes single letters and special characters from the analysis of Figure 1B-E as they might change the baseline.
+
+**We have added this analysis to our supplement.**
+
+> The word cloud of Figure 2B, C is somewhat nice as it shows the main words. However, this information could also be conveyed in the text. Would personally favor to quantitatively see loadings of first few principal components for different terms.
+
+**We have provided a table in our supplemental section.**
+
+> The definition of "True matches" could be more explicit in within the main text as the preceding figure 3A could for some people set up a different anticipation.
+
+**We agree this term is too vague, and we have updated the text to be more explicit.**
+
+```diff
+- Approximately 98% of our 200 pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were scored as true matches (Figure 3B).
+
++ Approximately 98% of our 200 pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were successfully matched with their published counterpart (Figure 2B)
+```
+
+> The association given in Figure 4A seems to mainly stem from a few papers with large distances. Would an association be present when using the rank-based Spearman correlation instead of a linear regression? Would, for visualization, a logarithmic relationship describe the data better than a linear one?
+
+**We believe there is a misconception about our figure. 
+This figure depicts the median half-life publication time for each preprint category within bioRxiv.**
```

Comment:
```suggestion
**This figure depicts the median half-life publication time for each preprint category within bioRxiv, so we don't think that it is driven by outliers within categories.
We have adjusted the figure legend to more clearly note what we plotted.**
```
Can you also adjust the first line of that figure legend to:
> Author-selected categories were associated with modest differences in the median time to publish.

Right now it being the median is in only the second sentence and might get overlooked.